src/lognormal_mixture_gibbs_eigen.cpp.bak

```diff
@@ -0,0 +1,620 @@
+// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; indent-tabs-mode: nil; -*-
```

Comment:
Se eu entendi certo isso aqui é equivalente ao outro arquivo do armadillo, mas usando o eigen, que você fez pra testar, mas que acabou voltando pro Armadillo certo?
Se sim, podemos deletar esse arquivo e comitar (o delete) para ele sumir?

Reply:
Exato, antes de especificar o **private(chain)** no **#pragma omp parallel for** ele estava encerrando a sessão do R por problemas, daí decidi investigar no Eigen se isso acontecia e lá descobri esse workaround de declarar a chain privada. Acabou que funcionou bem também no Armadillo e voltamos o código para ele por ser mais rápido.